libopenage/pathfinding/types.h

```diff
@@ -124,4 +125,19 @@ using cache_key_t = std::pair<portal_id_t, sector_id_t>;
  */
 using field_cache_t = std::pair<std::shared_ptr<IntegrationField>, std::shared_ptr<FlowField>>;
 
+/**
+ * Cost stamp for a given cost field cell.
+ */
```

Comment:
A small explanation of stamping for people who don't know what that is would be nice here. I added a suggesion below.

```suggestion
/**
 * Cost stamp for a cell on a cost field.
 * 
 * Stamps are used when cell costs are temporarily overwritten, e.g. when placing a game object
 * as an obstacle over terrain. Once the obstacle is removed, the cell can be reset to its original
 * value recorded by the stamp.
 */
```

Reply:
Hi @heinezen, sorry for the delay. Long time no see.

At what point should I stop templatising at Map::Map

Reply:
No Map::Map should not be templated I think. The templating should probably be constrained to the pathfinder system. But let's see how realistic that is...